Sources/AuthFoundation/User Management/Credential.swift

```diff
@@ -57,6 +57,8 @@ public class Credential: Identifiable, Equatable, OAuth2ClientDelegate {
     }
     
     /// Stores the given token for later use.
+    ///
+    /// This
```

Comment:
Is this supposed to be here?

Reply:
In the words of Scooby Doo, "Ruh Roh!"  I'll finish this comment and will push another commit.